addons/mail/models/mail_mail.py

```diff
@@ -138,10 +139,13 @@ def process_email_queue(self, ids=None):
                                 messages to send (by default all 'outgoing'
                                 messages are sent).
         """
+        # scheduled_date is in local time in db => user_now has to be in local time for comparison
+        user_tz = pytz.timezone(self.env.context.get('tz') or self.env.user.tz or 'UTC')
+        user_now = user_tz.fromutc(datetime.datetime.now())
         filters = ['&',
                    ('state', '=', 'outgoing'),
                    '|',
-                   ('scheduled_date', '<', datetime.datetime.now()),
+                   ('scheduled_date', '<', user_now.strftime('%Y/%m/%d %H:%M:%S')),
```

Comment:
Why is it different from DEFAULT_SERVER_DATETIME_FORMAT (containing DEFAULT_SERVER_DATE_FORMAT = "%Y-%m-%d" for the date part ) ?

Reply:
There is no good reason to use  '%Y/%m/%d %H:%M:%S' instead of DEFAULT_SERVER_DATETIME_FORMAT